packages/generators/src/handlers/default.ts

```diff
@@ -15,104 +15,68 @@ const resolveFile = (file: string): string => {
     if (devServer) {
-        self.dependencies = [...self.dependencies, 'webpack-dev-server'];
+        self.dependencies = [...self.dependencies, "webpack-dev-server"];
     }
 
     // Handle addition of html-webpack-plugin
-    const { htmlWebpackPlugin } = await Question.Confirm(
-        self,
-        'htmlWebpackPlugin',
-        'Do you want to simplify the creation of HTML files for your bundle?',
-        true,
-        self.force,
-    );
+    const { htmlWebpackPlugin } = await Question.Confirm(self, "htmlWebpackPlugin", "Do you want to simplify the creation of HTML files for your bundle?", true, self.force);
     if (htmlWebpackPlugin) {
-        self.dependencies = [...self.dependencies, 'html-webpack-plugin'];
+        self.dependencies = [...self.dependencies, "html-webpack-plugin"];
     }
 
     // Store all answers for generation
     self.answers = { ...self.answers, langType, devServer, htmlWebpackPlugin };
 
     // Handle CSS solutions
-    const { cssType } = await Question.List(
-        self,
-        'cssType',
-        'Which of the following CSS solutions do you want to use?',
-        ['none', 'CSS only', 'SASS', 'LESS', 'Stylus'],
-        'none',
-        self.force,
-    );
+    const { cssType } = await Question.List(self, "cssType", "Which of the following CSS solutions do you want to use?", ["none", "CSS only", "SASS", "LESS", "Stylus"], "none", self.force);
 
-    if (cssType == 'none') {
-        self.answers = { ...self.answers, cssType, isCSS: false, isPostCSS: false, extractPlugin: 'No' };
+    if (cssType == "none") {
+        self.answers = { ...self.answers, cssType, isCSS: false, isPostCSS: false, extractPlugin: "No" };
         return;
     }
 
-    const { isCSS } =
-        cssType != 'CSS only'
-            ? await Question.Confirm(self, 'isCSS', `Will you be using CSS styles along with ${cssType} in your project?`, true, self.force)
-            : { isCSS: true };
-
-    const { isPostCSS } = await Question.Confirm(
-        self,
-        'isPostCSS',
-        'Will you be using PostCSS in your project?',
-        cssType == 'CSS only',
-        self.force,
-    );
+    const { isCSS } = cssType != "CSS only" ? await Question.Confirm(self, "isCSS", `Will you be using CSS styles along with ${cssType} in your project?`, true, self.force) : { isCSS: true };
 
-    const { extractPlugin } = await Question.List(
-        self,
-        'extractPlugin',
-        'Do you want to extract CSS for every file?',
-        ['No', 'Only for Production', 'Yes'],
-        'No',
-        self.force,
-    );
+    const { isPostCSS } = await Question.Confirm(self, "isPostCSS", "Will you be using PostCSS in your project?", cssType == "CSS only", self.force);
+
+    const { extractPlugin } = await Question.List(self, "extractPlugin", "Do you want to extract CSS for every file?", ["No", "Only for Production", "Yes"], "No", self.force);
```

Comment:
This was more readable before.

Reply:
Weird, it should be breakable https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAExNwB4wBOAhmDAAoA2ArgOYCWUmAvpgLybEDux9WAitTgZ60AHQAZehgAUaOJQBmAGkwAdEPiKkKNBlA2qNAEQiYAnhGqYesTDDNaSZTAGEAyu8yKIhXADc4QnNveko4AH5DTABtDQA5CGiNAHkoShCfP3JCCAATajJRAxAjEABNYQ0AXTLE6PklMSywOABKAG5SkAgABxhitGRQYkJcrnJRhCGUYkoecyHlEAAjZwBrOBh3XtJGWmQiIWW4AFsVuDy8y4liKFpqYlo4ADFfU+IYAfvkEGJqBzdAAWMFOlAA6kC+MJdq13NM+PR-HxzL8wGgliBGPJCBQSLQPshFHN5MsAFZoPAAIQ2W3cxFOcCkUDgRJJcHJlPc+3Cggg8DZlFJIF2hBxvxWxAulG6vUIjBg4PoeRgQOQAA4AAzLOUQeTgki9X5y4RBQLdACO1H5cByfRmfzQAFoWZdLt1CHArfRPTknoSkMShRyQPJTvRDoRjqGeXA+QLA+zljApUqVWqkAAmZMkML7VwQU4BzRoACs3Wo8gAKlKZkHhf4hABJKDXWDuMDy-oAQVb7hg5nCgvkzGYQA

Reply:
yep it's only affecting typescript, maybe some eslint issue?

Reply:
Maybe, we need check it

Reply:
yep it's eslint, setting max width in there too

Reply:
weird, i.e. plugin don't read pretteir configuration?

Reply:
oh it doesn't pick default prettier rules but when we specify values it works, no need update eslint

Reply:
I've set it to 100, I think 100 makes sense

Reply:
80 is a bit too small considering types in params as well

Reply:
I think small is better more readable, reading all in one line is not good

Reply:
looks good in 100

Reply:
Can you provide line?

Reply:
lmk if something is messed up, 80 is a bit too small I feel which leads to over splitting code

Reply:
Maybe, but I want to look on examples :smile:

Reply:
yep, @snitin315 are you able to find anything else which looks odd?